engine/src/test/java/org/camunda/bpm/engine/test/bpmn/event/error/ErrorEndEventTest.java

```diff
@@ -69,5 +72,19 @@ public void testPropagateOutputVariablesWhileThrowError() {
     assertNotNull(runtimeService.getVariable(processInstanceId, "cancelReason"));
     assertEquals(42, runtimeService.getVariable(processInstanceId, "output"));
   }
+  
+  @Test
+  @Deployment
+  public void testErrorMessage() {

```

Comment:
@mboskamp, as in the Model API, I think we should have a test that covers the old behavior when no error message is defined.

Reply:
@koevskinikola You are right. I added a test for errors without message